### ISSUE TITLE
Submit to stackage at some point

### ISSUE THREAD

**Issue by epicallan** [mbj/conversions#14]

**Comment by mbj** [comment]
Yeah, I think its a good addition. Lets use it a bit more and propagate to some more projects before we do so?

**Comment by epicallan** [comment]
>Yeah, I think its a good addition. Lets use it a bit more and propagate to some more projects before we do so?

Sounds good.
